15_semana/aula5-projeto/src/Students/student.ts

```diff
@@ -0,0 +1,34 @@
+import { User } from "../Classes&Creators/user";
+import { UserCreator } from "../Classes&Creators/userCreator";
+import { FileManager } from "../FileManager";
+
+export class Student implements User, UserCreator{
+    public name: string
+    public email: string
+    public birthDate: string
+    protected mission: string
+
+    constructor(name: string, email: string, birthDate: string, mission: string){
+        this.name = name
+        this.email = email
+        this.birthDate = birthDate
+        this.mission = mission
+    }
+
+    public createUser(mission: string, name: string, email: string, birthDate: string){
```

Comment:
Esse método poderia ser estático.
Um método pode ser transformado em estático se você não utiliza nenhuma propriedade da classe. Perceba que aqui você não usou, em nenhum momento, a palavra `this`. Então esse poderia muito bem ser um método estático da classe `Student`